### PR TITLE
Dev layout

### PR DESCRIPTION
**Pull request by kiennguyenquoc**

### CODE REVIEW

**Comment by kiennguyenquoc** [comment]
using categories: [ruby on rails] ===> categories is "Ruby on rails"

File: _includes/nav_right.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          </ul>
+        </li>
+        <li><a href="/categories_articles/#web development">Web Development</a>
+          <ul>
+          {% for item in site.WebDev %}
+            <li><a href="/tags_articles_web/#{{ item }}">{{ item }}</a></li>
+          {% endfor %}
+          </ul>
+        </li>
+        <li><a href="/categories_articles/#infrastracrue">Infrastracrue</a>
```
Comment (vinh-nguyen) [inline _includes/nav_right.html]: @kiennguyenquoc infrastructure, not Infrastracrue

File: _includes/nav_right.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          <ul>
+          {% for item in site.Infrastracrue %}
```
Comment (vinh-nguyen) [inline _includes/nav_right.html]: @kiennguyenquoc infrastructure, not Infrastracrue

File: _includes/nav_right.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            <li><a href="/tags_articles_infrastracrue/#{{ item }}">{{ item }}</a></li>
```
Comment (vinh-nguyen) [inline _includes/nav_right.html]: @kiennguyenquoc infrastructure, not Infrastracrue

File: _includes/header.html [hunk 4]
```diff
@@ -1,27 +1,42 @@
...
+  <div class="wrapper-header">
+    <span>
+      <a class="site-title" href="http://zigexn.vn/" title="http://zigexn.vn/"><img alt="logo zigexn ventura" class="img-responsive" src="http://recruitment.zigexn.vn/img/logo.png"></a>
+      <a class="site-title" href="{{ site.baseurl }}/"><span class="header_title">{{ site.title }}</span></a>
+    </span>
+    <div class="dropdown">
+      <nav>
+        <ul class="clearfix">
+          <li><a href="/about/">About US</a>
+          <li><a href="">Categories</a>
```
Comment (vinh-nguyen) [inline _includes/header.html]: @kiennguyenquoc why this `a` tag has no `href`?

File: _includes/nav_right.html [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+        {% endfor %}
+      </ul>
+    </div>
+  </div>
+</div>
+
+<div class="follow">
+  <ul class="social-media-list">
+    <li>
+      Connect with us: <br><img class="icon_fb" src="https://upload.wikimedia.org/wikipedia/commons/thumb/c/c2/F_icon.svg/2000px-F_icon.svg.png" alt="logo-facebook"><a class="btn_fb" href="https://facebook.com/{{ site.facebook_username }}">
```
Comment (vinh-nguyen) [inline _includes/nav_right.html]: @kiennguyenquoc download this image to `/images` directory

**Pull request merged**